modules/jupyter_iap/charts/iap_jupyter/templates/backend-config.yaml

```diff
@@ -17,6 +17,14 @@ kind: BackendConfig
 metadata:
   name: {{ .Values.iap.backendConfig.name }}
 spec:
+  healthCheck:
```

Comment:
Why is this needed?

Reply:
for consistent healthcheck pass and quick healthly server status.